compose.c

```diff
@@ -515,28 +644,28 @@ static void update_crypt_info(struct ComposeRedrawData *rd)
   }
 #endif
 
-  redraw_crypt_lines(rd);
+  int row = calc_security(rd->email, &rd->sec_rows);
+  row += 2 + rd->to_rows + rd->cc_rows + rd->bcc_rows;
```

Comment:
Is this the same `2` used to initialize ` int used = 2;` ? If so, can we give it a name?

Reply:
It turns out that we don't need those lines any more.